handsontable/src/plugins/nestedRows/ui/headers.js

```diff
@@ -113,7 +117,8 @@ class HeadersUI extends BaseUI {
 
         if (ariaEnabled) {
           setAttribute(TH, [
-            A11Y_EXPANDED(false)
+            A11Y_EXPANDED(false),
+            A11Y_DESCRIPTION(this.hot.getTranslatedPhrase(ROW_HEADER_DESCRIPTION_EXPAND_ROW)),
```

Comment:
The expand/collapse button is hidden from the screen readers, so I added an instruction on how to expand/collapse the row with the keyboard shortcut.